src/Humanizer/DateHumanizeExtensions.cs

```diff
@@ -26,5 +26,86 @@ public static string Humanize(this DateTime input, bool utcDate = true, DateTime
 
             return Configurator.DateTimeHumanizeStrategy.Humanize(input, comparisonBase, culture);
         }
+
+        /// <summary>
+        /// Extracts the month out of a date
+        /// </summary>
+        /// <param name="input">The date of which you want the month</param>
+        /// <returns></returns>
+        public static string ToMonth(this DateTime input)
+        {
```

Comment:
Shouldn't it be [DateTimeFormatInfo.MonthNames](https://msdn.microsoft.com/en-us/library/system.globalization.datetimeformatinfo.monthnames) [DateTimeFormatInfo.MonthGenitiveNames](https://msdn.microsoft.com/en-us/library/system.globalization.datetimeformatinfo.monthgenitivenames)?